lib/logstash/filters/mutate.rb

```diff
@@ -266,9 +266,13 @@ def convert(event)
       if original.nil?
         next
       elsif original.is_a?(Hash)
-        @logger.debug("I don't know how to type convert a hash, skipping",
-                      :field => field, :value => original)
-        next
+        #@logger.debug("I don't know how to type convert a hash, skipping",
+        #              :field => field, :value => original)
+        #next
+        value = original
+        original.each do |hfield, hvalue|
```

Comment:
you can he each_pair here, check http://ruby-doc.org/core-2.2.0/Hash.html#method-i-each_pair